### PR TITLE
make mksdcard work for me

### PR DESCRIPTION
**Pull request by MichaelSp**

### CODE REVIEW

**Comment by gamaral** [comment]
Not sure what you want me to merge in... I'm guessing you did a pull-request by mistake?

**Comment by MichaelSp** [comment]
This was on purpose but my editor screwed the indention. I'll try again.

File: board/raspberrypi/mksdcard [hunk 1]
```diff
@@ -43,23 +43,23 @@ OUTPUT_PREFIX=""
 
 # check parameters
 
-if [ -z "${SDCARD}" ] || [ "${SDCARD}" == "-h" ] || [ "${SDCARD}" == "--help" ]; then
-	usage
-	exit 0
+if [ -z "${SDCARD}" ] || [ "${SDCARD}" = "-h" ] || [ "${SDCARD}" = "--help" ]; then
```
Comment (MichaelSp) [inline board/raspberrypi/mksdcard:46]: = instead of ==

**Comment by MichaelSp** [comment]
crap! Now the PR is completely screwed. I'll try again

**Pull request closed without merging**